src/stcal/ramp_fitting/utils.py

```diff
@@ -724,102 +724,10 @@ def output_integ(slope_int, dq_int, effintim, var_p3, var_r3, var_both3,
     return integ_info
 
 
-'''
-# BEGIN remove GLS
-def gls_output_integ(model, slope_int, slope_err_int, dq_int):
-    """
-    For the GLS algorithm, construct the output integration-specific results.
-    Parameters
-    ----------
-    model : instance of Data Model
-        DM object for input
-
-    slope_int : ndarray
-        Data cube of weighted slopes for each integration, 3-D float
-
-    slope_err_int : ndarray
-        Data cube of slope errors for each integration, 3-D float
-
-    dq_int : ndarray
-        Data cube of DQ arrays for each integration, 3-D flag
-
-    Returns
-    -------
-    cubemod : Data Model object
-    """
-    # Suppress harmless arithmetic warnings for now
-    warnings.filterwarnings("ignore", ".*invalid value.*", RuntimeWarning)
-    warnings.filterwarnings("ignore", ".*divide by zero.*", RuntimeWarning)
-
-    cubemod = datamodels.CubeModel()
-    cubemod.data = slope_int
-    cubemod.err = slope_err_int
-    cubemod.dq = dq_int
-
-    # Reset the warnings filter to its original state
-    warnings.resetwarnings()
-
-    cubemod.update(model)  # keys from input needed for photom step
-
-    return cubemod
-
-
-def gls_output_optional(model, intercept_int, intercept_err_int,
```

Comment:
I don't see (in an obvious way) that `gls_output_integ` and `gls_output_optional` are replicated anywhere else in the code, so are they no longer needed?

Reply:
They are no longer needed.  They were used when they code returned data models, but since the move to STCAL data models are no longer used internally, including as return values.